specification/containerregistry/data-plane/Microsoft.ContainerRegistry/preview/2019-08-15/containerregistry.json

```diff
@@ -736,7 +736,7 @@
             "description": "Repository attribute value",
             "required": false,
             "schema": {
-              "$ref": "#/definitions/ChangeableAttributes"
+              "$ref": "#/definitions/RepositoryChangeableAttributes"
```

Comment:
Curious as to why this is not consistent with other schema models like RepositoryAtrributeValue  just like TagAttributeValue

Reply:
Updated to use `parameters` instead of `Definition`. I tried to make minimal changes at first, but yes it makes more sense to be consistent.